rest/rest-api/src/main/java/org/ow2/proactive_grid_cloud_portal/common/SchedulerRestInterface.java

```diff
@@ -1957,4 +1957,22 @@ public JobIdData copyAndResubmitWithGeneralInfo(@HeaderParam("sessionid") String
     @Produces("application/json")
     Map<String, Object> getSchedulerPropertiesFromSessionId(@HeaderParam("sessionid")
     final String sessionId) throws NotConnectedRestException, PermissionRestException;
+
+    /**
+     * 
+     * Check if the user has the permission to execute the method passed as argument
+     * 
+     * @param sessionId
+     * @param jobId
+     * @param method
+     * @return true if the user has the permission to execute the java method
+     * @throws NotConnectedRestException
+     * @throws UnknownJobRestException
+     */
+    @GET
+    @Path("job/{jobid}/permission/{method}")
+    @Consumes(value = MediaType.APPLICATION_JSON)
+    @Produces("application/json")
+    boolean checkJobPermissionMethod(String sessionId, String jobId, String method)
+            throws NotConnectedRestException, UnknownJobRestException;
```

Comment:
![MINOR](https://sonarsource.github.io/sonar-github/severity-minor.png 'Severity: MINOR') Remove the redundant '!unknownSymbol!' thrown exception declaration(s). [![rule](https://sonarsource.github.io/sonar-github/rule.png)](https://sonarcloud.io/coding_rules#rule_key=squid%3ARedundantThrowsDeclarationCheck)